phase2/kubeadm/configure-vm-kubeadm-master.sh

```diff
@@ -48,62 +51,100 @@ networking:
   podSubnet: "$POD_NETWORK_CIDR"
 EOF
 
-# handle v1alpha1
-########################################################################
-if [[ "$KUBEADM_KUBERNETES_VERSION_MINOR" -lt "11" ]]; then
+  # add cloud provider
+  if [[ "$KUBEADM_ENABLE_CLOUD_PROVIDER" == true ]]; then
+    cat <<EOF |tee -a $KUBEADM_CONFIG_FILE
+apiServerExtraArgs:
+  cloud-provider: "$CLOUD_PROVIDER"
+EOF
+  fi
 
-  # add api version and token
-  cat <<EOF |tee -a $KUBEADM_CONFIG_FILE
+  # set ipvs
+  if [[ "$KUBEPROXY_MODE" == "ipvs" ]]; then
+    cat <<EOF |tee -a $KUBEADM_CONFIG_FILE
+kubeProxy:
+  config:
+    featureGates:
+      SupportIPVSProxyMode: true
```

Comment:
Why?

Reply:
the feature gate?
pre-1.11 the gate is needed.
in 1.11 this should be default to `true` as the gate is GA, although i didn't want to create a special branch to handle it.
pre-1.11 vs 1.11.